### DISCUSSION TITLE
Cannot read property 'HTML_STRING' of undefined

### DISCUSSION THREAD

**Discussion by kascigala** [jspsych/jsPsych#526] [Q&A]
I am currently trying to use `html-keyboard-response` plugin, and I get an error: `Uncaught TypeError: Cannot read property 'HTML_STRING' of undefined` on line 21.

Any ideas about a solution?

**Comment by jodeleeuw** [comment]
Are you loading the jspsych.js library before loading the plugin? Are you using the latest version of jspsych.js with the plugin?
